frontend/app/src/next/components/layouts/center-stage.layout.tsx

```diff
@@ -0,0 +1,23 @@
+import { cn } from '@/next/lib/utils';
+import { ReactNode } from 'react';
+
+interface CenterStageLayoutProps {
+  children: ReactNode;
+  className?: string;
+}
+
+export function CenterStageLayout({
+  children,
+  className = '',
+}: CenterStageLayoutProps) {
+  return (
+    <div
+      className={cn(
+        'min-h-screen w-full flex items-center justify-center',
+        className,
+      )}
+    >
+      <div className="w-full max-w-7xl px-4 sm:px-6 lg:px-8">{children}</div>
```

Comment:
I think this `max-w` is what Nick had reported with super wide screens not filling up right - maybe we should remove this?

Reply:
this component is specifically for things like the login page where we want it narrow